packages/server/src/games.ts

```diff
@@ -71,6 +71,20 @@ export async function getGame(id: string): Promise<GameResponse> {
   if (!game.players) {
     game.players = await BACKFILL_addEmptyPlayersArray(game);
   }
+  const playerClocks = Object.values(game.time_control?.forPlayer ?? {});
+  // bpj 2024-04-04: changed playerClock.remainingTimeMS to playerClock.clockState
```

Comment:
I just tested this part manually by starting a long Fischer game, editing the fields to look like an old game, then playing some moves.  It works fine, but leaves the old field behind, and never updates it.

<img width="438" alt="Screenshot 2024-04-05 at 9 15 04 PM" src="https://github.com/govariantsteam/govariants/assets/25233703/0ca22013-d41f-42b6-b4ad-a25443aa5618">

It's not concerning enough for me to try and mess with it more in this PR, but something to think about in the discussion in #225.